src/utils.ts

```diff
@@ -19,3 +21,64 @@ export function numberWithCommas(number: number) {
 }
 
 export const calcRate = (num1: number, num2: number) => (num1 / num2) * 100;
+
+export function useScroll() {
+  const [isScrolled, setIsScrolled] = useState(false);
+
+  useEffect(() => {
+    const handleScroll = _.debounce(() => {
+      setIsScrolled(!!window.scrollY);
+    }, 100);
+
+    window.addEventListener('scroll', handleScroll);
+
+    return () => window.removeEventListener('scroll', handleScroll);
+  });
+  console.log(isScrolled);
+
+  return isScrolled;
+}
+
+export function usePageInfo() {
+  const pages = [
+    {
+      path: '/create',
+      title: '여행 추가'
+    },
+    {
+      path: '/trip/[0-9]+/expense',
+      title: '쓴 돈 추가'
+    }
+  ];
+
+  const currentUrl = window.location.href;
+  const result = pages.find((p) => currentUrl.match(p.path));
```

Comment:
위의 코멘트에 부연 설명을 드리자면 path값이 currentUrl과 일치하는지 판별할때만 사용되는 것 같은데 아예 RegEx로 만들어서 test를 사용하는 것을 어떠신지 제안해봅니닷,,,

```suggestion
 const pages = [
   {
     pathRegEx: /\/create/,
     title: '여행 추가'
   },
   {
     pathRegEx: /\/trip\/[0-9]+\/expense/,
     title: '쓴 돈 추가'
    }
 ];
 const result = pages.find(page => page.pathRegEx.test(window.location.href));
```

Reply:
오 이런 방법도 있네요!! 깔끔하고 좋으네요 ^0^ 감사합니다 혜인님!